challenges/arrayShift/array-shift.js

```diff
@@ -0,0 +1,23 @@
+function insertShiftArray(arr, value) {
```

Comment:
looks good. you could clean it up like this if you wanted to but not needed. 
`function insertShiftArray(arr, value) {
  const newArray = [];
  const midIndex = Math.ceil((arr.length) / 2);
  for(let i = 0; i < arr.length + 1; i++){
    if(i < midIndex) newArray[i] = arr[i];
    
    if(i === midIndex) newArray[i] = value;
    
    if(i > midIndex) newArray[i] = arr[i - 1];
    
  }  

  return newArray;
}`